src/core/store/store.ts

```diff
@@ -52,3 +46,21 @@ function configureStore() {
 }
 
 export const store = configureStore();
+
+function patchTracer(t: TransactionData) {
+  const dump = t.actions
+    .filter((a) => !a.type.startsWith('invalidate '))
+    ?.map((a) => {
+      if (a.targets?.length === 1) {
+        const targ = '' + a.targets[0].id;
+        if (a.type.endsWith(targ)) {
+          const action = a.type.substring(0, a.type.length - targ.length - 1);
+          return targ + ' <' + action + '> ';
+        }
```

Comment:
```suggestion
        const target = String(a.targets[0].id)
        if (a.type.endsWith(target)) {
          const action = a.type.substring(0, a.type.length - target.length - 1);
          return target+ ' <' + action + '> ';
        }
```
if `'' + id` is for string creation is suggest this change